Makefile

```diff
@@ -436,8 +436,10 @@ benchmark_p2p_peerstore: ## Run P2P peerstore benchmarks
 
 ### Inspired by @goldinguy_ in this post: https://goldin.io/blog/stop-using-todo ###
 # TODO          - General Purpose catch-all.
+# ADR           - A TODO that will require a full on ADR in the future
```

Comment:
How do you feel about `DECIDE` or `DECISION` instead? I think totally makes sense to use `ADR` in the description but I suspect the todo itself would read nicer in context:

```
// DECIDE: whether to support X
...
// DECISION: how to optimize for Y
```